### PR TITLE
<ControlledCollapsedHeader/>

### PR DESCRIPTION
**Pull request by Wo1v3r**
### What changed
-  Added Controlled Collapse Header
-  Using it in <CollapsedHeader/> to avoid introducing breaking changes
-  Added Example
-  Added to Card documentation

### Why it changed
- We needed it controlled in Smart Actions Web

---

- [x] Change is tested
- [x] Change is documented
- [x] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: stories/GridWithCardLayout/ExampleControlledCollapsable.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              collapsed={this.state.collapsed}
+              handleCollapse={() => this.handleCollapse()}
+              title="Card with controlled collapsed header"
+              >
+              <Card.Content>
+                <Row>
+                  <Col span={12}>{renderStandardInput()}</Col>
+                </Row>
+              </Card.Content>
+            </Card.ControlledCollapsedHeader>;
```
Comment (shlomitc) [inline stories/GridWithCardLayout/ExampleControlledCollapsable.js]: redundant `;`

Comment (Wo1v3r) [reply stories/GridWithCardLayout/ExampleControlledCollapsable.js]: Ok

**Comment by Wo1v3r** [comment]
Not sure what you mean... 

Exposing a new component that wraps the CollapsedHeader in order to make it a controlled component..? 

I marked as tested meaning it does not break <CollapsedHeader> which is now using the controlled version.

I can make the example look better and tag for re-review.

**Comment by shlomitc** [comment]
I mean that `CollapsedHeader` is already managing the toggle state internally. You can just make it  "controllable" by passing a `value` and `onChange` props.
I prefer it over exposing `CollapsedHeaderControlled`

**Comment by Wo1v3r** [comment]
Of-course, I don't think that's the way to do it though. But I understand.
Happy Passover 👍

**Pull request closed without merging**